packages/maskbook/src/social-network-adaptor/twitter.com/injection/useEthereumName.ts

```diff
@@ -1,7 +1,7 @@
 import { useResolveEns } from '@masknet/web3-shared'
 import { useEffect, useMemo, useState } from 'react'
 
-const ENS_RE = /[\w#%+.:=@~-]{1,256}\.[\d()A-Za-z]{1,6}\b([\w#%&()+./:=?@~-]*)?/
+const ENS_RE = /[\w#%+.:=@~-]{1,256}\.(eth|zil|kred|xyz|luxe|crypto|nft|blockchain|bitcoin|coin|wallet|888|dao|x)\b/
```

Comment:
I think ENS is not supporting `crypto|nft|blockchain|bitcoin|coin|wallet|888|dao|x`. Instead, UNS supports them.